### ISSUE TITLE
Installing doesn't work because of unicode issue in README.rst

### ISSUE THREAD

**Issue by kristjanvariksoo** [pndurette/gTTS#115]
Tried to install using pip3 and pip. I'm using Arch Linux.

```
sudo pip3 install gTTS
Collecting gTTS
  Using cached https://files.pythonhosted.org/packages/eb/de/a932d79e20542a61649b80fc74798cce27677ba72507d55afa578f019379/gTTS-2.0.0.tar.gz
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-install-11a_6hmq/gTTS/setup.py", line 19, in <module>
        long_description=open('README.rst').read(),
      File "/usr/lib/python3.6/encodings/ascii.py", line 26, in decode
        return codecs.ascii_decode(input, self.errors)[0]
    UnicodeDecodeError: 'ascii' codec can't decode byte 0xc2 in position 1125: ordinal not in range(128)
    
    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in /tmp/pip-install-11a_6hmq/gTTS/
```

As a workaround I cloned the repo and edited the setup.py file to set `long_description` to a string instead of having it read from the file. This appears to of have worked as a workaround and it installed fine.

**Comment by pndurette** [comment]
Hmm! Not good. Thanks for that. Will try to replicate and look into it.

**Comment by mikozh** [comment]
got the same bug on OSX

**Comment by pndurette** [comment]
Alright thanks, I'll plan work on this then!

**Comment by pndurette** [comment]
@mikozh Can you let me know how you install on macOS? And versions, etc? I develop pretty much on macOS, and I've never seen that, but I want to replicate it. Thanks!

**Comment by pndurette** [comment]
Fixed in gTTS `2.0.1`!
